lib/jekyll/configuration.rb

```diff
@@ -26,7 +26,7 @@ class Configuration < Hash
       # Filtering Content
       'show_drafts'   => nil,
       'limit_posts'   => 0,
-      'future'        => true,           # remove and make true just default
```

Comment:
Oh interesting... it looks like we were going to remove it anyway. Looks like #690 is where we wrote that comment. I'm cool on keeping it.